modules/custom/az_event/config/install/core.entity_view_display.taxonomy_term.az_event_categories.az_badge.yml

```diff
@@ -13,23 +13,9 @@ third_party_settings:
     settings:
       smart_title__link: false
       smart_title__tag: div
-      smart_title__classes: {  }
-  field_group:
-    group_link:
-      children:
-        - smart_title
-      label: Link
-      parent_name: ''
-      region: content
-      weight: 0
-      format_type: link
-      format_settings:
-        classes: 'badge badge-light badge-link'
-        show_empty_fields: false
-        id: ''
-        target: entity
-        custom_uri: ''
-        target_attribute: default
```

Comment:
Now that I've updated the second view mode's id to `az_badge_alt` we probably don't need to update this.